contrib/linux/actions/checks/check_processes.py

```diff
@@ -52,7 +52,8 @@ def process(self, criteria):
                 cmdfh = open(self.procDir + "/" + p + "/cmdline")
                 cmd = cmdfh.readline()
                 pInfo[1] = cmd
-            except:
+            except IOError:
```

Comment:
Why specifically IOError? It is possible for other exception to occur in this block (i.e. index error, etc.) and this will skip the exception.